charts/amoro/templates/amoro-configmap.yaml

```diff
@@ -52,6 +52,7 @@ data:
           bind-port: {{ .Values.server.optimizing.port }}
 
       http-server:
+        rest-auth-type: {{ .Values.amoroConf.ams.restAuthType }}
```

Comment:
it is a config in AMS REST Serivce, so `rest-auth-type: {{ .Values.server.rest.restAuthType }}` is better option?

Reply:
@XBaith 
I currently see the `server.rest` variables in this Helm chart as being more related to Kubernetes service or ingress configurations rather than application settings. On the other hand, I consider the configurations under `ams.` to be application-specific, which is why I placed them there. I also think this might be a part of the chart worth refactoring in the future..

Since there are differing opinions on this, I wanted to share my perspective. Could you please provide some guidance on this PR?

Reply:
The reason I prefer moving them under `server.rest` is that we already have the property `HTTP_SERVER_REST_AUTH_TYPE` configured as part of the HTTP REST service. It might be better to align with the original design. What do you think?

Reply:
Hi @baiyangtx, please let me know if you have any questions.
I will merge this if there is no more comments.